### ISSUE TITLE
802.1x Authenticator and Authentication Server impersonation attacks

### ISSUE THREAD

**Issue by sophron** [sophron/wifiphisher#127]
We can target 802.1X networks by implementing an Authenticator and an Authentication server that will always return an EAP-Success message so that the client believes she is connected to their legitimate authenticator. We can mount our phishing attacks as usual afterwards.

The paper <a href="http://users.suse.com/~lnussel/The_Evil_Twin_problem_with_WPA2-Enterprise_v1.1.pdf">here</a> explains the attack in more detail.

**Comment by anakin1028** [comment]
@sophron The link for the paper seems not available for now?

**Comment by sophron** [comment]
Yes, looks like it. But there other available resources online. Here is one: https://pentest.blog/attacking-wpa-enterprise-wireless-network/ (WPA Enterprise section)

**Comment by sophron** [comment]
Attacks in Enterprise WLANs is one of the things that we should focus in the upcoming release.

**Comment by javier-es** [comment]
@sophron so, wifiphisher supports now attacks to AP using 802.1x? In #128 I saw that roguehostapd was implemented, but I don't know if that means that 802.1x/WPA2-EAP is supported.
